README.md

```diff
@@ -1 +1,55 @@
-# To-do-React-App
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+#  Math-Magicians React App
+
+> "Math-Magicians"  is a webapp for all fans of mathematics. It is a Single-Page-App (SPA) that allows users to make simple calcualtions and read a randoom amth-related quote. This Single-Page-app was created using React
```

Comment:
Your ReadMe file is for a Math-Magician React App, but this repo is a Todo React App.
Kindly update your ReadMe file to reflect this repository as this will make your repository more informative and professional. You can check out this [template](https://github.com/microverseinc/readme-template) and edit it to fit your project.